src/report/js/ReportBuilder.js

```diff
@@ -1499,22 +1500,24 @@ define([
           yearObject['color'] = hexColor;
         },
 
+        // Create the Progression Chart
         getFireCounts: function () {
           const self = this;
           const queryFor = self.currentISO ? self.currentISO : 'global';
           const handleAs = {handleAs: 'json'};
           const promiseUrls = [];
 
           if (window.reportOptions.aoiId) {
+            // Queries to use if the report is for a country's subregions
             const urls = [
               `${Config.fires_api_endpoint}admin/${queryFor}?aggregate_values=True&aggregate_time=month&fire_type=modis&period=2001-01-01,${moment().utcOffset('Asia/Jakarta').format("YYYY-MM-DD")}`,
               `${Config.fires_api_endpoint}admin/${queryFor}/${window.reportOptions.aoiId}?aggregate_values=True&aggregate_time=month&fire_type=modis&period=2001-01-01,${moment().utcOffset('Asia/Jakarta').format("YYYY-MM-DD")}`,
             ];
             promiseUrls.push(...urls);
           } else {
+            // Queries to use if the report is for a country-wide or global report
             const urls = [
-              `${Config.fires_api_endpoint}admin/${queryFor}?aggregate_values=True&aggregate_time=month&fire_type=modis&period=2001-01-01,${moment().utcOffset('Asia/Jakarta').format("YYYY-MM-DD")}`,
-              `${Config.fires_api_endpoint}admin/${queryFor}?aggregate_values=True&aggregate_time=month&aggregate_admin=adm1&fire_type=modis&period=2001-01-01,${moment().utcOffset('Asia/Jakarta').format("YYYY-MM-DD")}`
```

Comment:
Taking this call out only gives us the One request! Check out what happens after our `Promise.all` and what it does with the `length` of our `responses` array!